MidasCivil_Adapter/Convert/ToMidasCivil/Properties/FromMetadata.cs

```diff
@@ -0,0 +1,92 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Text;
+using BH.oM.Adapters.MidasCivil;
+using System.Collections.Generic;
+
+namespace BH.Adapter.Adapters.MidasCivil
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static StringBuilder FromMetadata(Metadata metadata)
+        {
+            StringBuilder midasMetadata = new StringBuilder();
```

Comment:
We should replace any `=` as you can cause an error if any of the fields contain when loading the .mct file.

Also, we should remove any line breaks in the strings, otherwise you end up with a scenario like this:
;DESCRIPTION= This description
occupies two lines

Especially for something like the description where you can see a user having a separate line.